store/LocalStore.js

```diff
@@ -2,14 +2,22 @@
 
 const lru = require('tiny-lru')
 
-function LocalStore (timeWindow, cache) {
-  this.lru = lru(cache || 5000)
-  setInterval(this.lru.clear.bind(this.lru), timeWindow).unref()
```

Comment:
This `setInterval` made this module extremely fast. Can you please restore this?
We should not be be starting a timer for each URL, it's very expensive.

Reply:
100% agree on this point, maybe a global clock could fix the issue?
lìLike the one implemented in [easy-breaker](https://github.com/delvedor/easy-breaker#caveats).